README.md

```diff
@@ -0,0 +1,188 @@
+This demo shows how to migrate a Spring Boot application to Quarkus, and how to measure the performance gains along the way. It can all be live-coded, showing how easy migrating can be.
+
+## Preparation
+
+The supporting scripts for this demo assume MacOS. You will need the `http` utility:
+
+```shell
+brew install httpie
+```
+
+You will also need a container runtime, such as Podman or Docker.
+
+Finally, you will need `jq`. 
+
+## The Spring Boot Application 
+
+
+Start with `quarkus-spring-todo-app`. This is the [Quarkus todo sample](https://github.com/cescoffier/quarkus-todo-app), rewritten to Spring Boot 3.3.0. There is a single test implemented using Test Containers and Rest Assured (using Rest Assured isn't strictly idiomatic Spring, but it makes things easier later on.) For examples of migrating tests, see [Eric Deandrea](https://github.com/edeandrea)'s comprehensive "Spring to Quarkus" [code samples](https://github.com/quarkus-for-spring-developers/examples). 
+
+### JVM mode
+
+Package it and launch it:
+
+```
+cd spring-todo-app
+./mvnw package
+```
+
+Before going further, start the database that the live instance will be using. Once that's done, start the app.
+
+```
+just start-infra
+java -jar target/spring-todo-app-0.0.1-SNAPSHOT.jar -Xmx600m -Xms600m
```

Comment:
600 seems like a lot. I generally do it with 256m

Reply:
Yeah, I had to guess! I was hoping you'd share your actual values. :)